pkgs/development/libraries/qt-6/modules/qtbase.nix

```diff
@@ -79,7 +86,7 @@
 , dconf
 , gtk3
   # options
-, libGLSupported ? true
+, libGLSupported ? !stdenv.isDarwin
```

Comment:
Shouldn't this be stdenv.isLinux instead?

Reply:
IMO if libGL is an explicitly Linux-specific thing, yes; if libGL is a thing on other platforms but not Darwin, no.

Reply:
Done.